tools/download.R

```diff
@@ -2,10 +2,26 @@ jq_cdn_download <- function(version) {
   basenames <- paste0("jquery-", version, c(".min.js", ".min.map", ".js"))
   target <- file.path("inst", "lib", version)
   dir.create(target, recursive = TRUE)
-  download.file(
-    file.path("https://code.jquery.com", basenames),
-    file.path(target, basenames)
+  Map(
+    basename = basenames,
+    f = function(basename) {
+      download.file(
+        file.path("https://code.jquery.com", basename),
+        file.path(target, basename)
+      )
+    }
   )
+
+  # Add in source map location
+  # Required given comments in https://blog.jquery.com/2014/01/24/jquery-1-11-and-2-1-released/
+  jquery_min_js <- file.path(target, paste0("jquery-", version, ".min.js"))
+  # Point to the version-less source map file
+  cat(
+    file = jquery_min_js,
+    append = TRUE,
+    paste0("//# sourceMappingURL=jquery-", version, ".min.map\n")
+  )
```

Comment:
Should we throw or something if `sourceMappingURL` already exists in `jquery_min_js`?

Reply:
[593ccf5](https://github.com/rstudio/jquerylib/pull/14/commits/593ccf5e097fefc3dbaa79843fd12d0fdf69c34b)